Makefile

```diff
@@ -74,3 +77,19 @@ dist: brupop-image
 clean:
 	-rm -rf target
 	rm -f -- '$(DISTFILE)'
+
+check-crd-golden-diff:
+	# Visualizes diff between the CRD template in our helm chart vs the generated golden file
+	# from the rust definitions. This is useful to ensure there are no hanging changes
+	# that need to be made to the template.
+	diff --color \
+		deploy/charts/bottlerocket-shadow/templates/custom-resource-definition.yaml \
+		deploy/src/golden/custom-resource-definition.yaml || return 0
+
+manifest:
+	echo --- > bottlerocket-update-operator.yaml && \
+	kubectl create namespace brupop-bottlerocket-aws \
+		--dry-run=client \
```

Comment:
Does this require a kubeconfig or Kubernetes cluster? If not, this is cool...

Reply:
Yes, it just generates the yaml for us. For example, here, I unset my context (so I'm not targeting any cluster) and I can still generate the yaml:

```
❯ k config unset current-context
Property "current-context" unset.

❯ kubectl create namespace test-namespace --dry-run=client -o yaml
apiVersion: v1
kind: Namespace
metadata:
  creationTimestamp: null
  name: test-namespace
spec: {}
status: {}
```